Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -292,12 +292,18 @@ public void setSelectedPhysicalNode(TreeNode selectedPhysicalNode) {
     }
 
     Optional<LogicalDivision> getSelectedStructure() {
+        if (Objects.isNull(selectedLogicalNode)) {
+            return null;
```

Comment:
According to Coding Guidelines returning `null` value should be avoided. There are a few places in the source code where an additionl null check must be added and not only at the one place later in this pull request.

According to my IDE (IntelliJ IDEA) the return statement here should be written as `return Optional.empty()`.